index.js

```diff
@@ -1,17 +1,61 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [],
+    this.length = 0
+  }
 
-  add(item) {}
+  add(item) {
+      this.items.push(item);
+      this.items.sort((a, b) => a - b);
+      this.length ++;
```

Comment:
this is correct, but as a stylistic choice, I like to not change the value, but reassign it.

```js
this.length = this.items.length
```